cylc/flow/cfgspec/globalcfg.py

```diff
@@ -300,17 +348,33 @@
             Conf('handlers', VDR.V_STRING_LIST)
             Conf('handler events', VDR.V_STRING_LIST)
             Conf('mail events', VDR.V_STRING_LIST)
-            Conf('startup handler', VDR.V_STRING_LIST)
-            Conf('timeout handler', VDR.V_STRING_LIST)
-            Conf('inactivity handler', VDR.V_STRING_LIST)
-            Conf('shutdown handler', VDR.V_STRING_LIST)
-            Conf('aborted handler', VDR.V_STRING_LIST)
-            Conf('stalled handler', VDR.V_STRING_LIST)
-            Conf('timeout', VDR.V_INTERVAL)
-            Conf('inactivity', VDR.V_INTERVAL)
-            Conf('abort on timeout', VDR.V_BOOLEAN)
-            Conf('abort on inactivity', VDR.V_BOOLEAN)
-            Conf('abort on stalled', VDR.V_BOOLEAN)
```

Comment:
Note some unplanned config name changes here. I had to distinguish between workflow and stall timeout on this branch, as discussed, so I went slightly further and used consistent naming and grammar for all the timeout and abort items. E.g. `abort on inactivity` really means abort if the inactivity timer times out, hence `abort on inactivity timeout` c.f. `abort on stall timeout` etc.

Reply:
I'll add them here.